src/plugins/data/public/search/search_interceptor/search_interceptor.ts

```diff
@@ -434,7 +435,7 @@ export class SearchInterceptor {
     }
   );
 
-  private showRestoreWarningToast = (sessionId?: string) => {
+  private showRestoreWarningToast = (_sessionId?: string) => {
```

Comment:
why the underscores ?

Reply:
These are unused variables that cause warnings in my editor. 

I'll leave it alone and handle the noise locally. I'm sure at some point it would be easier to just increase the lint rules, and automate these fixes.

Reply:
i think the underscores actually violates our code style guide ?